apps/web/src/pages/Users/AdminApplicantProfilePage/AdminApplicantProfilePage.tsx

```diff
@@ -56,51 +62,147 @@ interface AdminApplicantProfileProps {
 const AdminApplicantProfile = ({ query }: AdminApplicantProfileProps) => {
   const intl = useIntl();
   const user = getFragment(AdminApplicantProfile_Fragment, query);
+  const [openSections, setOpenSections] = useState<string[]>([]);
+  const hasOpenSections = openSections.length > 0;
+
+  const toggleSections = () => {
+    const newValue = hasOpenSections ? [] : Object.values(SECTION_KEY);
+    setOpenSections(newValue);
+  };
 
   return (
     <Container className="my-18">
       <TableOfContents.Wrapper>
         <TableOfContents.Navigation>
           <TableOfContents.List>
             <TableOfContents.ListItem>
-              <TableOfContents.AnchorLink id={PERSONAL_CONTACT_INFO_ID}>
+              <TableOfContents.AnchorLink id="basic-information">
+                {intl.formatMessage({
+                  defaultMessage: "Basic information",
```

Comment:
This heading section has a larger font-size and heavier font-weight than other `h2` in other tabs: @Jerryescandon is this on purpose?

Reply:
Good catch, yeah should be the same weight and size across all tabs.